src/executors/ruby-mysql.yml

```diff
@@ -2,14 +2,15 @@ parameters:
   ruby_version:
     description: version of Ruby
     type: string
-    default: latest
+    default: "3.2"
   db_version:
     description: version of MySQL
     type: string
     default: "8.0"
 docker:
-  - image: circleci/ruby:<< parameters.ruby_version >>-node-browsers
+  - image: cimg/ruby:<< parameters.ruby_version >>-browsers
     environment:
       DATABASE_ADAPTER: mysql2
   - image: cimg/mysql:<< parameters.db_version >>
-    command: mysqld --default-authentication-plugin=mysql_native_password
+    command:
+      ["mysqld",  "--default-authentication-plugin=mysql_native_password", "--character-set-server=utf8mb4"]
```

Comment:
On `circleci/mysql`, `mysqld` uses following character set:

```sql
MySQL [circle_test]> SHOW GLOBAL VARIABLES LIKE 'character%';
+--------------------------|----------------------------+
| Variable_name            | Value                      |
+--------------------------|----------------------------+
| character_set_client     | utf8                       |
| character_set_connection | utf8                       |
| character_set_database   | utf8                       |
| character_set_filesystem | binary                     |
| character_set_results    | utf8                       |
| character_set_server     | utf8                       |
| character_set_system     | utf8                       |
| character_sets_dir       | /usr/share/mysql/charsets/ |
+--------------------------|----------------------------+
8 rows in set (0.003 sec)
```

On default `cimg/mysql`, following:

```sql
mysql> SHOW VARIABLES LIKE 'character%';
+--------------------------|----------------------------+
| Variable_name            | Value                      |
+--------------------------|----------------------------+
| character_set_client     | latin1                     |
| character_set_connection | latin1                     |
| character_set_database   | latin1                     |
| character_set_filesystem | binary                     |
| character_set_results    | latin1                     |
| character_set_server     | latin1                     |
| character_set_system     | utf8                       |
| character_sets_dir       | /usr/share/mysql/charsets/ |
+--------------------------|----------------------------+
8 rows in set (0.00 sec)
```

So, we should set `character-set-server`.

But `utf8` is deprecated by https://dev.mysql.com/doc/refman/8.0/en/charset-unicode-sets.html :

> utf8: An alias for utf8mb3. In MySQL 8.0, this alias is deprecated; use utf8mb4 instead. utf8 is expected in a future release to become an alias for utf8mb4.

And [our `generate-database_yml` command uses `utf8mb4` (not `utf8`)](https://github.com/agileware-jp/redmine-plugin-orb/blob/v2.3.0/src/commands/generate-database_yml.yml#L22). So we must set `utf8mb4`, I think.